advanced_tutorials/bitcoin/streamlit_app.py

```diff
@@ -113,8 +116,10 @@ def get_latest_date():
     latest_unix = feature_view.query.read().unix.max()
```

Comment:
Why do you do this? this will take lot of time to read entire data and then find latest timestamp. Just use fv.get_batch_data(start_time = "...", end_time= "..."). You can you timedelta to find out start and end time from now